backend/src/main/kotlin/rocks/didit/sefilm/database/entities/Showing.kt

```diff
@@ -23,6 +24,7 @@ data class Showing(@Id
                    val admin: UserID = UserID(),
                    val payToUser: UserID = admin,
                    val expectedBuyDate: LocalDate? = null,
-                   val participants: Set<UserID> = setOf(),
+                   val participants: Set<Participant> = setOf(),
```

Comment:
Nice! We should probably never do `Set<UserID>` and similar since it makes migrations a _PITA_.  If we used an object Like `Participant` instead, adding new fields (as long as they're nullable) is trivial.